package.json

```diff
@@ -69,6 +69,7 @@
     "dayjs": "^1.11.13",
     "dayjs-plugin-utc": "0.1.2",
     "debounce": "^2.1.0",
+    "dotenv": "^16.4.5",
```

Comment:
Do we need dotenv now? we can use the inbuilt functionality of nodejs now https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs